packages/sandbox/src/sandbox.ts

```diff
@@ -37,6 +37,13 @@ export type SandboxOptions = {
   format?: ClientConfigFormat;
   profile?: string;
   watchForChanges?: boolean;
+  functionStreamingOptions?: SandboxFunctionStreamingOptions;
```

Comment:
related to a comment above, I would remove this from the SandboxOptions and instead inject it directly into the LambdaFunctionLogStreamer